samples/nrf5340/netboot/README.rst

```diff
@@ -7,82 +7,126 @@ nRF5340: Network core bootloader
+* Enabling the :option:`CONFIG_SECURE_BOOT` option in the application that has to run on the nRF5340 network core.
+  The build system will include this sample automatically.
+* Enabling the :option:`CONFIG_BOOTLOADER_MCUBOOT` and :option:`CONFIG_BT_RPMSG_NRF53` options in the application that has to run on the application core.
+  The build system will enable the :option:`CONFIG_SECURE_BOOT` option automatically for the network core application.
 
-Testing
-=======
+When you include this image in a multi-image build, the build system generates a new set of firmware update files
+These files match the ones described in :ref:`mcuboot:mcuboot_ncs`, except that they contain the network core application firmware and are prefixed with ``net_core_``.
+
+.. |sample path| replace:: :file:`samples/nrf5340/netboot/`
+
+.. include:: /includes/build_and_run.txt
+See also the following pages:
+
+* :ref:`ug_nrf5340_building` for additional details on building samples on the nRF5340.
+* :ref:`configure_application` for additional details on configuring your application.
+
+See below for an example of a multi-image build that includes :ref:`peripheral_uart`, using either SEGGER Embedded Studio or the command line:
+
+.. tabs::
 
-To test the network core bootloader sample run the following commands:
+   .. group-tab:: SEGGER Embedded Studio
 
-#. |connect_terminal|
+      Follow the instructions linked above, enabling the following options:
 
-   Note that on the nRF5340 DK has multiple UART instances, so the correct port must be identified.
+      * :option:`CONFIG_BOOTLOADER_MCUBOOT`:
 
-#. ``west build -b nrf5340dk_nrf5340_cpuapp -d build_netboot samples/bluetooth/peripheral_uart -f -- -DCONFIG_BOOTLOADER_MCUBOOT=y -Dmcuboot_CONFIG_PCD=y``
+        #. Select :guilabel:`Project` -> :guilabel:`Configure nRF Connect SDK project`.
+        #. Go to :guilabel:`Modules` -> :guilabel:`Nordic nRF Connect` -> :guilabel:`Bootloader` and set :guilabel:`Use MCUboot` to enable :option:`CONFIG_BOOTLOADER_MCUBOOT`.
+        #. Click :guilabel:`Configure`.
 
-   This sample (:file:`samples/bluetooth/peripheral_uart`) will automatically include the network core sample ``hci_rpmsg`` when built for the nRF5340 DK.
-   Since we enable MCUboot (``-DCONFIG_BOOTLOADER_MCUBOOT=y``) in the application core, the network core bootloader is automatically included as well.
-   In addition to this, it is necessary to enable the :ref:`subsys_pcd` subsys for the MCUboot image (``-Dmcuboot_CONFIG_PCD=y``).
+      * :option:`mcuboot_CONFIG_PCD`:
 
-#. ``west flash -d build_netboot``
+        #. Select :guilabel:`Project` -> :guilabel:`Configure nRF Connect SDK project`.
+        #. Go to :guilabel:`Modules` -> :guilabel:`Nordic nRF Connect` -> :guilabel:`Peripheral CPU DFU (PCD)` and set :guilabel:`Use Peripheral CPU DFU (PCD)` to enable :option:`mcuboot_CONFIG_PCD`.
```

Comment:
```suggestion
        #. Go to :guilabel:`Modules` -> :guilabel:`Nordic nRF Connect` -> :guilabel:`Peripheral CPU DFU (PCD)` and select :guilabel:`Peripheral CPU DFU Support (PCD)` to enable :option:`CONFIG_PCD`.
```

Reply:
Not sure why this shows up different for me. Please test.

Reply:
This option is already enabled for me. So maybe better to say that users should make sure it is enabled?

Reply:
fixed